src/http-client.js

```diff
@@ -142,7 +142,8 @@ function parseHeaderValues(headers) {
   return parsedHeaders;
 }
 
-function buildRequest(input, init = {}) {
+function buildRequest(input, init) {
+  init || (init = {});
```

Comment:
@vbauerster Assigning default values like in the line below isn't more conventional? Besides, is more used throughout the framework. @bryanrsmith